flashlight/nn/modules/PrecisionCast.cpp

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Please remove the copyright header and license from files that have them. Is there a reason you've added them?

Reply:
>> workplace.